### PR TITLE
[CBRD-23960] Support Generated Keys

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-23960

please refer to the demo code for getGeneratedKeys() functionality for INSERT queries: test18() and test19() in demo/test/TestCUBRIDStatement.java

### CODE REVIEW

File: src/method/method_query_handler.cpp [hunk 1]
```diff
@@ -235,6 +235,217 @@ namespace cubmethod
...
+    if (error == NO_ERROR)
+      {
+	/* set qresult_info */
+	query_result_info &result_info = info.qresult_info;
+	result_info.stmt_type = stmt_type;
+	result_info.tuple_count = info.generated_keys.tuples.size ();
+	result_info.query_id = -1; /* initialized value, intead of garbage */
+      }
+
+    return info;
```
Comment (beyondykk9) [inline src/method/method_query_handler.cpp:282]: error is set but not used.

Comment (hgryoo) [reply src/method/method_query_handler.cpp:282]: if local variable, `error` is set NO_ERROR, safely able to set qresult_info (line 273 ~ 280). Errors are handled by m_error_ctx and passed to callback_handler.

**Review by shparkcubrid — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Review by ctshim — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

File: src/jsp/com/cubrid/jsp/impl/SUStatement.java [hunk 2]
```diff
@@ -154,6 +155,24 @@ public SUStatement(
         isSensitive = false;
     }
 
+    public SUStatement(SUConnection conn, GetGeneratedKeysInfo info) {
+        type = GET_AUTOINCREMENT_KEYS;
```
Comment (hyunikn) [inline src/jsp/com/cubrid/jsp/impl/SUStatement.java:161]: This and two other SUStatements constructors do not set suConn field with the conn argument.
Is it OK?

Comment (hgryoo) [reply src/jsp/com/cubrid/jsp/impl/SUStatement.java:161]: I've fixed. thank you.

**Review by hyunikn — APPROVED** [review]

**Pull request merged**